frontend/packages/dev-console/src/components/monitoring/alerts/MonitoringAlerts.tsx

```diff
@@ -0,0 +1,252 @@
+import * as React from 'react';
+import Helmet from 'react-helmet';
+// FIXME upgrading redux types is causing many errors at this time
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { useSelector, useDispatch } from 'react-redux';
+import { match as RMatch } from 'react-router-dom';
+import {
+  expandable,
+  Table,
+  TableHeader,
+  TableBody,
+  sortable,
+  SortByDirection,
+} from '@patternfly/react-table';
+import { rules } from './rules';
+import './MonitoringAlerts.scss';
+import * as _ from 'lodash';
+import { FilterToolbar } from '@console/internal/components/filter-toolbar';
+import { getURLSearchParams, ResourceLink, Kebab } from '@console/internal/components/utils';
+import { fromNow } from '@console/internal/components/utils/datetime';
+import {
+  Severity,
+  StateCounts,
+  AlertState,
+  getAlertsAndRules,
+  PrometheusRule,
+} from '@console/internal/components/monitoring';
+import { monitoringSetRules } from '@console/internal/actions/ui';
+import { RootState } from '@console/internal/redux';
+import { PodModel } from '@console/internal/models';
+import { alertDescription, alertState } from '@console/internal/reducers/monitoring';
+
+type MonitoringAlertsProps = {
+  match: RMatch<{
+    ns?: string;
+  }>;
+};
+
+enum AlertStates {
+  firing = 'Firing',
+  pending = 'Pending',
+  silenced = 'Silenced',
+  notFiring = 'Not Firing',
```

Comment:
The `Not Firing` state was removed from the admin console, so shouldn't be needed here either.